### PR TITLE
Update 2022-02-09-this-week-in-rust.md

### PR DESCRIPTION
**Pull request by kw-fn**

### CODE REVIEW

File: draft/2022-02-09-this-week-in-rust.md [hunk 1]
```diff
@@ -19,14 +19,21 @@ If you find any errors in this week's issue, [please submit a PR](https://github
...
 ### Newsletters
 
 ### Research
 
 ### Observations/Thoughts
+* [Some mistakes Rust doesn't catch](https://fasterthanli.me/articles/some-mistakes-rust-doesnt-catch)
 
 ### Rust Walkthroughs
+* [Containerless! How to Run WebAssembly Workloads on Kubernetes with Rust](https://developer.okta.com/blog/2022/01/28/webassembly-on-kubernetes-with-rust)
+* [`curl` with Rust](https://daniel.haxx.se/blog/2022/02/01/curl-with-rust/)
```
Comment (nellshamrell) [inline draft/2022-02-09-this-week-in-rust.md:36]: Looks like we got this one in a previous pull request. Do you mind removing it?

**Review by nellshamrell — CHANGES_REQUESTED** [review]

File: draft/2022-02-09-this-week-in-rust.md [hunk 2]
```diff
@@ -19,14 +19,20 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 ### Foundation
 
 ### Project/Tooling Updates
+* [Nushell v0.44](https://www.nushell.sh/blog/2022-02-08-nushell_0_44.html)
+* [Rust API for the Firefox Profiler](https://blog.mozilla.org/performance/2022/02/08/rust-api-for-the-firefox-profiler/)
+* [Starship v1.3.0](https://github.com/starship/starship/releases/tag/v1.3.0)
```
Comment (nellshamrell) [inline draft/2022-02-09-this-week-in-rust.md:24]: While this update certainly looks very cool, I don't feel there is enough content beyond the list of changes to feature it in the newsletter (I'm looking for more content about how to use the project, how it helps someone do cool things with Rust, etc.) My fellow editors and I are working on some better guidelines for project updates and will be publishing them within the next few weeks. I will take this link out before I merge this PR.

**Review by nellshamrell — APPROVED** [review]
Ty!

File: draft/2022-02-09-this-week-in-rust.md (hunk 2 above)
Comment (kw-fn) [reply draft/2022-02-09-this-week-in-rust.md]: Good idea. Thanks!

**Pull request merged**